mesh/mesh.hpp

```diff
@@ -1358,8 +1360,11 @@ class Mesh
        @sa GetBdrElementAdjacentElement() */
    void GetBdrElementAdjacentElement2(int bdr_el, int &el, int &info) const;
 
-   /// Return the local face index for the given boundary face.
-   int GetBdrFace(int BdrElemNo) const;
+   /// @brief Return the local face index for the given boundary face.
```

Comment:
For the given boundary element no?

Reply:
Fixed in c794dae53b1d3723a4d1b0a39aa576d848a2a79e